video/cloud-client/analyze/analyze.py

```diff
@@ -122,26 +140,65 @@ def analyze_labels(path):
     # first result is retrieved because a single video was processed
     results = operation.result().annotation_results[0]
 
-    for i, label in enumerate(results.label_annotations):
-        print('Label description: {}'.format(label.description))
-        print('Locations:')
+    # Process video/segment level label annotations
```

Comment:
This seems like a pretty long snippet, is that intended ?

Reply:
Yes indeed - we often group the basic use cases of different methods into one script.